tsl/src/bgw_policy/job.c

```diff
@@ -770,7 +786,11 @@ job_execute(BgwJob *job)
 	if (!ActiveSnapshotSet())
 	{
 		pushed_snapshot = true;
+#if (PG12 && PG_VERSION_NUM >= 120800) || (PG13 && PG_VERSION_NUM >= 130400) || PG14_GE
```

Comment:
```suggestion
#if (PG12 && PG_VERSION_NUM >= 120008) || (PG13 && PG_VERSION_NUM >= 130004) || PG14_GE
```
Middle bits of numeric version are not used past 9.6
```
dev=# select current_setting('server_version_num'),version();
 current_setting |                                   version
-----------------+------------------------------------------------------------------------------
 130003          | PostgreSQL 13.3 on x86_64-pc-linux-gnu, compiled by gcc (GCC) 11.1.0, 64-bit
(1 row)
```

Reply:
Ooops ... my bad ... fixed!